### PR TITLE
[Improvement][API-Test] Add api tests for project page

### LINKED ISSUE #14401: [Improvement][API-Test] Add api test cases for project page
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* Add api test cases for project page
* related: #10411

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Add api tests for project page
* Fix the path of test cases from `xxx/api.test/xxx` to `xxx/api/test/xxx`.
* This PR closes: #14401 

## Brief change log

* See above.

## Verify this pull request

* API test cases successfully passed.

### CODE REVIEW

File: dolphinscheduler-ui/pnpm-lock.yaml [hunk 1]
```diff
@@ -1,165 +1,108 @@
...
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-lockfileVersion: '6.0'
+lockfileVersion: 5.3
+
```
Comment (EricGao888) [inline dolphinscheduler-ui/pnpm-lock.yaml]: restore changes to this file

File: dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/resources/workflow-json/test.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+[ {
+  "processDefinition" : {
+    "id" : 1,
+    "code" : 9752686452032,
+    "name" : "test",
+    "version" : 1,
+    "releaseState" : "OFFLINE",
```
Comment (EricGao888) [inline dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/resources/workflow-json/test.json]: remove this

File: dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/utils/MultipartUtility.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
+package org.apache.dolphinscheduler.api.test.utils;
+
```
Comment (EricGao888) [inline dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/utils/MultipartUtility.java]: remove this file

**Review by kezhenxu94 — DISMISSED** [review]

File: dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/cases/ProjectAPITest.java [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@DolphinScheduler(composeFiles = "docker/basic/docker-compose.yaml")
+@Slf4j
+// TODO: Add more detailed permission control related cases after userPage test cases completed
+public class ProjectAPITest {
```
Comment (SbloodyS) [inline dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/cases/ProjectAPITest.java:47]: Please add this to the https://github.com/apache/dolphinscheduler/blob/9c92b4b240a30e54464509b2e696e3310894507a/.github/workflows/api-test.yml#L90-L94

**Review by SbloodyS — CHANGES_REQUESTED** [review]

File: dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/cases/ProjectAPITest.java (hunk 4 above)
Comment (EricGao888) [reply dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/cases/ProjectAPITest.java:47]: done, thx

**Review by SbloodyS — APPROVED** [review]
+1

**Pull request merged**